### PR TITLE
update dubbogo to 1.5 latest commit

### PR DESCRIPTION
**Pull request by cityiron**
change to 1.5 branch commit https://github.com/apache/dubbo-go/commit/9c3bd0e9c92d70698f08f0ed4f81c8369a8346b6 to auto test by github-action

### CODE REVIEW

**Comment by cityiron** [comment]
Run success. So I think the latest of dubbogo's 1.5 branch is available in the scenario of automatic test examples.

File: go.sum [hunk 1]
```diff
@@ -100,17 +100,25 @@ github.com/alecthomas/units v0.0.0-20190717042225-c3de453c63f4/go.mod h1:ybxpYRF
 github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d/go.mod h1:rBZYJk541a8SKzHPHnH3zbiI+7dagKZ0cgpgrD7Fyho=
 github.com/alibaba/sentinel-golang v0.6.2 h1:1OjjpljJbNKWp9p5RJKxOqS1gHGZPUWPlCcokv5xYJs=
 github.com/alibaba/sentinel-golang v0.6.2/go.mod h1:5jemKdyCQCKVf+quEia53fo9a17OSe+wnl9HX2NbNpc=
+github.com/alibaba/sentinel-golang v1.0.2 h1:Acopq74hOtZN4MV1v811MQ6QcqPFLDSczTrRXv9zpIg=
```
Comment (AlexStocks) [inline go.sum:103]: pls using command `go mod tidy` to delete redundant libs

Comment (cityiron) [reply go.sum:103]: OK

**Review by zouyx — APPROVED** [review]

**Review by beiwei30 — APPROVED** [review]
LGTM

**Pull request merged**